.github/workflows/bitbar-prepare-artifacts.yaml

```diff
@@ -46,23 +46,23 @@ jobs:
       - name: List build tools versions
         run: ls /Users/runner/Library/Android/sdk/build-tools/
 
-      # Sign auth-debug-androidTest.apk
-      - name: Sign auth-debug-androidTest.apk
+      # Sign app-debug-androidTest.apk
+      - name: Sign app-debug-androidTest.apk
         uses: r0adkll/sign-android-release@v1
         with:
-          releaseDirectory: samples/auth/build/outputs/apk/androidTest/debug
+          releaseDirectory: samples/app/build/outputs/apk/androidTest/debug
           signingKeyBase64: ${{ secrets.SIGNING_KEYSTORE }}
           alias: ${{ secrets.SIGNING_ALIAS }}
           keyStorePassword: ${{ secrets.SIGNING_KEYSTORE_PASSWORD }}
           keyPassword: ${{ secrets.SIGNING_KEY_PASSWORD }}
         env:
           BUILD_TOOLS_VERSION: "34.0.0"
 
-      # Sign forgerock-auth-debug-androidTest.apk
-      - name: Sign forgerock-auth-debug-androidTest.apk
+      # Sign forgerock-integration-tests-debug-androidTest.apk
+      - name: Sign forgerock-integration-tests-debug-androidTest.apk
```

Comment:
The e2e tests are now in the `forgerock-integration-tests` module... This module includes all e2e tests (`forgerock-core`, `forgerock-auth`, `ping-protect`, `forgerock-authenticator`)